examples/src/main/java/io/opencensus/examples/helloworld/QuickStart.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2016-17, OpenCensus Authors
```

Comment:
-18

Btw, you can only put the current year.

Reply:
Done.